pandas-stubs/core/series.pyi

```diff
@@ -184,6 +187,8 @@ from pandas.core.dtypes.dtypes import CategoricalDtype
 
 from pandas.plotting import PlotAccessor
 
+_T_COMPLEX = TypeVar("_T_COMPLEX", bound=complex)
```

Comment:
I don't understand why you need a `TypeVar` here.  Why not just use `complex` wherever you have `_T_COMPLEX` ?

Reply:
ok